index.html

```diff
@@ -513,6 +519,28 @@ <h2 id="collections">Collection Functions (Arrays or Objects)</h2>
 =&gt; [4, 5, 2, 3, 0, 1]
 </pre>
 
+<p id="transform">
+        <b class="header">transform</b><code>_.transform(collection, iteratee, [memo], [context])</code>
+        <br />
+        <tt>_.transform</tt> is an alternative to <tt>_.reduce</tt> that transforms
+        the given collection into a new collection. Unlike <tt>_.reduce</tt> if the
+        memo isn't specified it will default to a new instance of the type of the collection.
```

Comment:
We've discussed why its more flexible than reduce in some cases in the thread it was discussed #1901, it's also usually less verbose. Theres a number of places it will be extremely useful for `Marionette` in cases where there is a contract `{*} => {*} | [*] => [*]` (see my `deepMap` or [`deepPick`](http://codereview.stackexchange.com/a/58279/32103) examples)